samples/DurationPickerSample.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+	<title>Duration Picker Sample</title>
+	<!-- -->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<!-- Less.js (uncomment for client-side rendering of less stylesheets; leave commented to use only CSS)
+	<script src="../../../enyo/tools/minifier/node_modules/less/dist/less-1.7.0.min.js"></script> -->
```

Comment:
Please update less related routine. You need to refer https://github.com/enyojs/moonstone/blob/master/samples/ButtonSample.html